inbc-program/inbc/parser/source_app_parser.py

```diff
@@ -35,8 +35,8 @@ def application_add(args: argparse.Namespace) -> str:
 
     manifest += '<repo><repos>'
 
-    for source in args.sources:
-        manifest += '<source_pkg>' + source + '</source_pkg>'
+    source_tags = [f'<source_pkg>{source}</source_pkg>' for source in args.sources]
```

Comment:
**Suggestion:** Use a more efficient way to concatenate strings for 'manifest' by using ''.join() method, which is faster and more idiomatic than using '+' repeatedly. [performance]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
manifest_parts = [
    '<?xml version="1.0" encoding="utf-8"?>',
    '<manifest>',
    '<type>source</type>',
    '<applicationSource>',
    '<add>'
]
manifest = ''.join(manifest_parts)
```